tests/test_gmm.py

```diff
@@ -297,6 +298,9 @@ def test_cuda(self, test_case_description, mixture_count, class_count, features,
         # Ensure result are as expected
         np.testing.assert_allclose(results, expected, atol=1e-3)
 
+    def test_load(self):
+        load_module("gmm", {"CHANNEL_COUNT": 2, "MIXTURE_COUNT": 2, "MIXTURE_SIZE": 3}, verbose_build=True)
```

Comment:
Hi @charliebudd, I'm trying to add some module loading tests, but getting this error
from ubuntu without GPU https://github.com/Project-MONAI/MONAI/runs/6261951047?check_suite_focus=true, any suggestions? logs:
```py
======================================================================
ERROR: test_load (tests.test_gmm.GMMTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/MONAI/MONAI/tests/test_gmm.py", line 302, in test_load
    load_module("gmm", {"CHANNEL_COUNT": 2, "MIXTURE_COUNT": 2, "MIXTURE_SIZE": 3}, verbose_build=True)
  File "/home/runner/work/MONAI/MONAI/monai/_extensions/loader.py", line 86, in load_module
    module = load(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/torch/utils/cpp_extension.py", line 1144, in load
    return _jit_compile(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/torch/utils/cpp_extension.py", line 1382, in _jit_compile
    return _import_module_from_library(name, build_directory, is_python_module)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/torch/utils/cpp_extension.py", line 1775, in _import_module_from_library
    module = importlib.util.module_from_spec(spec)
  File "<frozen importlib._bootstrap>", line 556, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 1166, in create_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
ImportError: /tmp/tmppx9wx2hi/gmm_2_2_3_Linux_3_8_12_111/gmm_2_2_3_Linux_3_8_12_111.so: undefined symbol: _Z10apply_cudaPKfS0_Pfjj

```